tests/testthat/test-utils_review.R

```diff
@@ -1,15 +1,15 @@
 test_that("md_to_word generates expected Word doc", {
-  docx_file <- "review.docx"
+  is_ci <- testthat:::on_ci()
```

Comment:
Curios, why do we check the testing environment?

Reply:
This code is on an outdated commit. How are you reviewing it? Always use Files Changed tab, I think you may be looking under Commits

![image](https://github.com/nhsbsa-data-analytics/nhsbsaShinyR/assets/91202154/12999394-7005-45b8-b5e1-d3de236383a2)

Why it was used at one point tho - GitHub Actions runs on Windows Server, not windows 10. This was making the exact content of the Word doc slightly different than it would be when run locally. So a slightly different test would be applied on GHA than would be applied locally.

I got rid of needing to do different tests by instead setting the sensitivity of the test to be enough for it to pass on GHA. Since the difference in content is very small, this is adequate even when run locally.